### PR TITLE
CATTY-531 Store detail view landscape mode

### PR DESCRIPTION
**Pull request by v15a1**
* Changed data type of `descriptionLabel` to a UILabel
* Added constraints to resize the description to horizontal/vertical orientation width

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by m-herold — CHANGES_REQUESTED** [review]
Constraints look good, however please change back to ```
`ActiveLabel` to support links (see CATTY-497).

**Comment by v15a1** [comment]
> Constraints look good, however please change back to 
> `ActiveLabel` to support links (see CATTY-497).

Will do!

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift [hunk 1]
```diff
@@ -114,23 +113,25 @@ let kHTMLAHrefTagPattern = "href=\"(.*?)\""
...
-            descriptionLabel.frame = CGRect(x: (view?.frame.size.width ?? 0.0) / 15, y: height * 0.35 + 40, width: 540, height: expectedSize.height)
-        } else {
-            descriptionLabel.frame = CGRect(x: (view?.frame.size.width ?? 0.0) / 15, y: height * 0.35 + 40, width: 280, height: expectedSize.height)
-        }
+        let descriptionLabel = ActiveLabel(frame: .zero)
+        descriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+        view?.addSubview(descriptionLabel)
+
+        NSLayoutConstraint.activate([
+            descriptionLabel.leadingAnchor.constraint(equalTo: view!.leadingAnchor, constant: (view?.frame.size.width ?? 0) / 15),
```
Comment (m-herold) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:121]: Please avoid force-unwrappings.

**Review by m-herold — CHANGES_REQUESTED** [review]
Looks good, only one suggestions. Please squash your commits afterwards. Thanks a lot!

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift [hunk 2]
```diff
@@ -81,15 +81,14 @@ let kHTMLAHrefTagPattern = "href=\"(.*?)\""
         self.setMaxHeightIfGreaterFor(view, withHeight: authorLabel.frame.origin.y + authorLabel.frame.size.height)
     }
 
-    private func addProjectDescriptionLabel(withDescription description: String, to view: UIView?, target: Any?) -> CGFloat {
+    private func addProjectDescriptionLabel(withDescription description: String, to view: UIView!, target: Any?) -> CGFloat {
```
Comment (m-herold) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:84]: Please avoid force unwrappings.

**Review by m-herold — CHANGES_REQUESTED** [review]

**Review by m-herold — APPROVED** [review]
Thanks @v15a1!

**Pull request merged**